shell/components/ResourceDetail/Masthead.vue

```diff
@@ -101,6 +101,14 @@ export default {
   },
 
   computed: {
+    dev() {
+      try {
+        return this.$store.getters['prefs/get'](PLUGIN_DEVELOPER);
+      } catch {
+        return this.$store.getters['prefs/get'](DEV);
+      }
```

Comment:
can we push this logic into the getter. I have a feeling this pattern could be used in multiple places.

Reply:
Done